.config/forest.dic

```diff
@@ -30,3 +30,7 @@ tipset/SM
 tipsetkey/S
 tuple
 validator/S
+serializing
+SignedMessage
+Blake2b256
+deserializing
```

Comment:
```suggestion
de-serializing
```

Reply:
"deserializing" is valid/commonly used, but we can add "de-serializing" as well if it will be frequently used.

Reply:
It is a bit arbitrary but `deserialize` isn't in the dictionary we use. We could add it to our custom list of words that means more work later as we have to write a definition for all of them.